scripts/github-changelog.sh

```diff
@@ -19,3 +19,5 @@ echo "- Generating changelog (${OUTPUT_FILE})"
 changelog-gen -changelog "${CHANGELOG_TEMPLATE}" \
 	-owner "${GH_ORG}" -repo "${GH_REPO}" \
 	"${RELEASE_SHA}" "${MASTER_SHA}" | tee "${OUTPUT_FILE}"
+
+echo "**Docker Image Perma Link**: ghcr.io/${GH_ORG}/${GH_REPO}:${RELEASE_SHA}" >> ${OUTPUT_FILE}
```

Comment:
I think this is now duplicate and should be removed...

Reply:
my mistake removed